HPE3PARPSToolkit-CLI.psm1

```diff
@@ -7882,7 +7884,7 @@ Function New-3parGroupVVCopy
 	}
 	else
 	{
-		return "Please Select Atlist one from P R or Halt"
+		return "Please Select At-least One from P R or Halt"
```

Comment:
Change the return string as "To proceed, select either PR or Halt"
But looks like this sentence is in complete. Lets discuss it before finalising the sentence

Reply:
Updated.